compiler/parser/parser.peg

```diff
@@ -46,21 +46,24 @@
 #define VAR(x) let x
 #endif
 
-start = __ ast:Sequential __ EOF { RETURN(ast) }
+start
+  = __ scope:Scope __ EOF { RETURN(scope) }
+  / __ seq:Sequential __ EOF { RETURN(seq) }
 
-Sequential
-  = decls:Decls __ first:Operation rest:SequentialTail* {
-      RETURN(MAP("kind": "Sequential", "ops": PREPEND(first, rest), "decls": decls))
+Scope
+  = decls:Decl+ seq:Sequential {
+      RETURN(MAP("kind": "Scope", "body": seq, "decls": decls))
```

Comment:
I'm not sure what you want me to do.  Change scope.Body to scope.Sequential?  Or change body:Sequential to seq:Sequential and then go with "body":seq?  To me it's most clear that the seq is in the body of a scope so I prefer it as I've written it and in the next PR body will be an ast.Seq which is a []dag.Op rather than a struct.

Reply:
No, I'm just suggesting changing the name `seq` to `body` here.